init.lua

```diff
@@ -72,7 +68,9 @@ function toolranks.new_afteruse(itemstack, user, node, digparams)
   if lastlevel < level then
     local levelup_text = "Your " .. toolranks.colors.green ..
                          itemdesc .. toolranks.colors.white ..
-                         " just leveled up!"
+                         " just leveled up to " ..
+                         toolranks.colors.green .. "level " ..
```

Comment:
I don't think we need the level part at all; player will probably know which level his tool is.